src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for better readability and maintainability. Typically, external dependencies are imported first, followed by internal dependencies. In this case, 'prop-types' (an external dependency) should be imported before '@react-navigation/core' (an internal dependency).
```diff
- import PropTypes from 'prop-types';
- import {useRoute} from '@react-navigation/core';
+ import {useRoute} from '@react-navigation/core';
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->